.github/workflows/push.yml

```diff
@@ -0,0 +1,37 @@
+name: 'Continuous Integration'
+on:
+  push:
+    branches-ignore:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+jobs:
+  test:
+    runs-on: ubuntu-16.04
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v2-beta
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: 'npm install, build, and test'
+      run: |
+        npm install -g npm
```

Comment:
@orangejulius do you remember why this step is nessesary?

Reply:
Compatibility fix between node v0.12 and v4+ ?

Reply:
I don't think it matters today, but at times we have wanted to make sure we are on the latest version of NPM to get features like `npx`, etc.

Node.js releases are much better at staying up to date with NPM these days, so it might be safe to remove if we really wanted to.

Reply:
I've removed it but open to adding it back if required.